README.md

```diff
@@ -30,6 +30,10 @@ The default values for the HTTP server deamon are `httpd` (used by Apache) for R
 
 (RedHat/CentOS only) If you have enabled any additional repositories (might I suggest [geerlingguy.repo-epel](https://github.com/geerlingguy/ansible-role-repo-epel) or [geerlingguy.repo-remi](https://github.com/geerlingguy/ansible-role-repo-remi)), those repositories can be listed under this variable (e.g. `remi-php56,epel`). This can be handy, as an example, if you want to install the latest version of PHP 5.6, which is in the Remi repository.
 
+    php_repostate: "installed"
```

Comment:
I think I was confused a bit by the variable name here—can we change it to `php_packages_state` instead? `repostate` makes it sound like it would only affect repos... whereas `packages_state` makes it more directly refer to the state of the installed PHP packages.